### PR TITLE
Saves last directory in text file and sets file openers to that directory

### PR DESCRIPTION
**Pull request by MStraatman**

### CODE REVIEW

File: src/main/java/window/FileSelector.java [hunk 1]
```diff
@@ -53,10 +61,52 @@ public static FileChooser getInstance() {
...
+            saveDirectory(chosenFile.getParentFile().getAbsolutePath());
         }
         return chosenFile;
     }
+
+    /**
+     * Finds the last used directory from which a file was chosen.
+     * @return the path of the directory.
+     */
+    private static String openDirectory() {
```
Comment (jjtiessen) [inline src/main/java/window/FileSelector.java]: This method name suggests this method will open a directory while it returns a directory path. Consider refactoring it.

Comment (MStraatman) [inline src/main/java/window/FileSelector.java]: True, I renamed the method to clarify what it's for

**Comment by MStraatman** [comment]
I cannot seem to find another working alternative. I tried to make it work with the Preferences API, but there seems to be a bug with denied access and I cannot find a fix. There are more people on the internet with the same problem, but those fixes don't work.

File: src/main/java/window/FileSelector.java [hunk 2]
```diff
@@ -1,9 +1,19 @@
 package window;
 
+import filesystem.FileSystem;
 import javafx.stage.FileChooser;
```
Comment (tvheems) [inline src/main/java/window/FileSelector.java]: If you group the import, group all of the imports with each other. Now the two javafx imports are separated.

Comment (MStraatman) [inline src/main/java/window/FileSelector.java]: Fixed

**Pull request merged**